### PR TITLE
Minimal APIs support

### PR DESCRIPTION
**Pull request by doddgu**
# Description

added AddDapr extension method for DaprEndpointRouteBuilderExtensions to support minimal APIs. And added tests for this, but I only tested invocation manually, I'm not sure the other features are working.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #695

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by davidfowl** [comment]
See my comment, I don't think any of this is required.

File: src/Dapr.AspNetCore/DaprEndpointRouteBuilderExtensions.cs [hunk 1]
```diff
@@ -21,6 +25,39 @@ namespace Microsoft.AspNetCore.Builder
...
     public static class DaprEndpointRouteBuilderExtensions
     {
+        /// <summary>
+        /// Adds Dapr integration for Minimal APIs to the provided <see cref="IEndpointRouteBuilder" />.
+        /// </summary>
+        /// <param name="builder">The <see cref="IEndpointRouteBuilder" />.</param>
+        /// <param name="services">The <see cref="IServiceCollection" />.</param>
+        /// <param name="configureClient">The (optional) <see cref="DaprClientBuilder" /> to use for configuring the DaprClient.</param>
+        /// <returns>The <see cref="IEndpointRouteBuilder" /> builder.</returns>
+        public static IEndpointRouteBuilder AddDapr(this IEndpointRouteBuilder builder, IServiceCollection services, Action<DaprClientBuilder> configureClient = null)
```
Comment (rynowak) [inline src/Dapr.AspNetCore/DaprEndpointRouteBuilderExtensions.cs:35]: The design of this does not really make sense to me. I would expect the **services** to already be baked by the time you have an endpoint route builder.

Comment (rynowak) [inline src/Dapr.AspNetCore/DaprEndpointRouteBuilderExtensions.cs:35]: The design of this does not really make sense to me. I would expect the **services** to already be baked by the time you have an endpoint route builder. 

It's still not super clear to me what (if any) changes are needed and what scenarios are being enabled. Let's use the issue to discuess.

**Comment by rynowak** [comment]
Closing in favor of #738

**Pull request closed without merging**